### DISCUSSION TITLE
Allow copying of output of jupyter / interactive window

### DISCUSSION THREAD

**Discussion by nickeubank** [microsoft/vscode-jupyter#9124] [New Ideas]
Is there a reason one can't copy the output of a jupyter or interactive cell? 

I often print out diagnostics, and it's very frustrating to not be able to just highlight the output and paste it in slack/email.

**Comment by nickeubank** [comment]
ah apparently this SHOULD work... https://github.com/microsoft/vscode-jupyter/issues/767

Will try and figure scope of when usually not working to post issue...

**Comment by greazer** [comment]
What type of output are you trying to copy? I.e., copying standard output from a cell like:
```
print('hello')
```
works. However, some output types do not... say a plot from Plotly.

**Reply by greazer** [reply]
More than likely you're running into this: https://github.com/microsoft/vscode/issues/129880.

**Reply by nickeubank** [reply]
No, I have intermittent issues with standard output. I'll try and isolate when it happens...

**Reply by greazer** [reply]
Thanks @nickeubank. If you find the scenario, please considering entering a new issue here: https://github.com/microsoft/vscode-jupyter/issues/new/choose
